FINSApp/src/finsUDP.c

```diff
@@ -2611,7 +2627,7 @@ static asynStatus ReadFloat64(void *pvt, asynUser *pasynUser, epicsFloat64 *valu
 
 /* send FINS request */
 
-	if (finsSocketRead(pdrvPvt, pasynUser, (void *)&value_f, 2, addr, NULL, sizeof(float)) < 0)
+	if (finsSocketRead(pdrvPvt, pasynUser, (void *)&value_f, 1, addr, NULL, sizeof(float)) < 0)
```

Comment:
I changed it as it was causing a memory overwrite and crash in debug mode, the function `ReadFloat64` only calls 32bit fins reads, so reading two items into  value_f is an overwrite. If it were to do a 16bit read then I agree it would need to have 2 here, but currently it only does 32bit reads.

I can't recall if I tested on emulators as I had tested the branch against real hardware, both larmor and helium recovery plcs, so i could check float readbacks and other things. The helium recovery plc is the only db that actually uses float32 so this particular change should i believe not affect any other tests.

Thanks for checking, I will run them myself and see what is wring